product/views.py

```diff
@@ -1,2 +1,70 @@
+import json
+
+from django.http    import HttpResponse
+from django.http    import JsonResponse
+from django.views   import View
+
+from product.models import Category, DetailImage, Product
+
+
+
+
+
+#class ListView(View):
+#    def get(self, request):
+#
+#        categories = Category.objects.all()
+#
+#        nav_list = [{
+#            'category_id' : nav.id,
+#            'name' : nav.name
+#        } for nav in categories]
+
+
+class MainView(View):
+
+    def get(self, request):
+
+        try :
+            categories = Category.objects.all()
+
+            item_list = [{
+                'product_id' : item.id,
+                'image' : item.detailimage_set.first().image_url,
+                'name' : item.name,
+                'short_info' : item.short_description
+            } for item in categories]
```

Comment:
database에서 category 리스트를 가져오기 때문에 product_id가 아니라 category_id가 맞지 않나요?
제가 이해한게 맞나요 예진님??
```suggestion
            results = [{
                'category_id' : category.id,
                'image' : category.detailimage_set.first().image_url,
                'name' : category.name,
                'short_info' : category.short_description
            } for category in categories]
```

Reply:
네..ㅎㅎㅎ..맞아요..... 수정했습니다!